packet_buffer.go

```diff
@@ -11,15 +11,17 @@ import (
 // packetBuffer represents a packet buffer
 type packetBuffer struct {
 	packetSize       int
+	filter           PacketFilter
```

Comment:
Could you rename this variable from `filter` to `s` ?

Reply:
Done